CMakeLists.txt

```diff
@@ -111,54 +111,50 @@ if (${run_unittests} OR ${run_int_tests})
     add_subdirectory(tests)
 endif()
 
-if(${use_installed_dependencies})
-    # Set CMAKE_INSTALL_* if not defined
-    include(GNUInstallDirs)
+# Set CMAKE_INSTALL_* if not defined
+include(GNUInstallDirs)
 
-    if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
-        set(CMAKE_INSTALL_LIBDIR "lib")
-    endif()
+if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+    set(CMAKE_INSTALL_LIBDIR "lib")
+endif()
+
+# Install UMock C
+set(package_location "cmake")
+
+install(TARGETS umock_c EXPORT umock_cTargets
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}/../bin
+    INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
+)
+install(FILES ${umock_c_h_files} "./inc/umock_c/umock_c_prod.h" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/umock_c)
 
-    # Install UMock C
-    set(package_location "cmake")
+include(CMakePackageConfigHelpers)
 
-    install(TARGETS umock_c EXPORT umock_cTargets
-        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-        RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}/../bin
-        INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
-    )
-    install(FILES ${umock_c_h_files} "./inc/umock_c/umock_c_prod.h" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/umock_c)
```

Comment:
>install(FILES ${umock_c_h_files} "./inc/umock_c/umock_c_prod.h" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/umock_c) [](start = 4, length = 112)

This line is no longer required because the umock_c_prod.h sits alongside other headers under

Reply:
Fixed.